### ISSUE TITLE
eFP gene linkouts

### ISSUE THREAD

**Issue by jd-campbell** [legumeinfo/microservices#615]
@sammyjava 

On SoyBase Classic Webpage we have a specific uBAR figure on [gene model pages](https://soybase.org/sbt/search/search_results.php?category=FeatureName&version=Glyma2.0&search_term=Glyma.15g026400). 
This figure has the expression pattern for that specific gene model within multiple tissue types. 
The image can be seen under the Expression Patterns banner and I tool a screenshot of the uBAR figure I am referring to.  We are currently bringing these figures using the uBAR API. We basically we bring in the figures by the subsequent URL ([example](https://bar.utoronto.ca/api/efp_image/efp_soybean/soybean_severin/Absolute/Glyma.15g026400)).

**Our question for you:** Is there a way to add these figures to the gene model pages on GlycineMine?

![Screen Shot 2024-01-18 at 14 34 03](https://github.com/legumeinfo/mine-issues/assets/13021601/9e0a3a27-d609-40ab-b390-2a59109891a6)

**Comment by adf-ncgr** [comment]
Yes, I think your reasoning about making this a linkout is sensible, at least as an initial way of connecting things. There is some charm to having the eFP images embedded in the page as done at ThaleMine, but as you rightly noted it would not work for the vast majority of genes in our system so probably better to make it a linkout for those genes they support. I've just added one for glyma.Wm82.gnm4.ann1 genes though it's not immediately obvious to me whether they are using gnm2.ann1 or gnm4.ann1. I'll figure it out later, but let me know if you think I'm pointing to what you had in mind when you requested this, @jd-campbell
